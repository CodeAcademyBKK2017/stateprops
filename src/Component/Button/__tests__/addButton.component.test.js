import React from 'react';
import addButton from '../addButton.component';
import renderer from 'react-test-renderer';

test('addButton Snapshot Test',() => {
  let text = 'Add';
  const tree = renderer.create(<addButton onClick={jest.fn()}>{text}</addButton>);
  expect(tree).toMatchSnapshot();
});
