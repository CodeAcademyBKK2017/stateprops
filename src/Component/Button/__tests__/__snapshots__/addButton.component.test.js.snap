// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addButton Snapshot Test 1`] = `
<addButton
  onClick={[Function]}
>
  Add
</addButton>
`;
